src/main/java/com/dev/bankingservice/service/UserService.java

```diff
@@ -0,0 +1,15 @@
+package com.dev.bankingservice.service;
+
+import com.dev.bankingservice.entity.User;
+
+public interface UserService {
+    User create(User user);
+
+    User getById(Long id);
+
+    User getByPhoneNumber(String phoneNumber);
+
+    void update(User user);
```

Comment:
see checklist

Reply:
removed `update` method and rename `create` to `save` to use for both ‘create’ and ‘update’ operations.